spark/src/test/scala/geotrellis/spark/summary/StatsTileRDDMethodsSpec.scala

```diff
@@ -113,7 +113,9 @@ class StatsTileRDDMethodsSpec extends AnyFunSpec with TestEnvironment with TestF
       hist.merge(hist2).quantileBreaks(70) should be (hist.quantileBreaks(70))
     }
 
-    it ("should be able to sample a fraction of an RDD to compute a histogram") {
+    // TODO: fix this test before merge
+    // 30000.0 was not less than or equal to 20000.0 (StatsTileRDDMethodsSpec.scala:128)
+    ignore ("should be able to sample a fraction of an RDD to compute a histogram") {
```

Comment:
Probably this is caused by the spire upgrade.

Reply:
Lets move it into a separate issue https://github.com/locationtech/geotrellis/issues/3379